### PR TITLE
dev_Yunduo

### PR DESCRIPTION
**Pull request by YunduoLuo**
this feature is for :)

### CODE REVIEW

File: lab-exercise-1/YunduoFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class YunduoFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (nickttrps) [inline lab-exercise-1/YunduoFeature.java:3]: fighting!

File: lab-exercise-1/YunduoFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class YunduoFeature {
+      public void feature() {
```
Comment (tnktian) [inline lab-exercise-1/YunduoFeature.java:2]: yeah man!!! continued!!!

File: lab-exercise-1/YunduoFeature.java (hunk 1 above)
Comment (sRubyy) [inline lab-exercise-1/YunduoFeature.java:3]: Well done bro keep fighting ^^

**Comment by arthasem** [comment]
ヾ(*'▽'*)

File: lab-exercise-1/YunduoFeature.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
 public class YunduoFeature {
       public void feature() {
           // this is a new feature
+          int x = 0;
+          x = x+1;
```
Comment (nickttrps) [inline lab-exercise-1/YunduoFeature.java:5]: susususu.

Comment (tnktian) [inline lab-exercise-1/YunduoFeature.java:5]: >< you're so good laa

**Review by plphxwxt — COMMENTED** [review]
Good job

**Pull request merged**